scripts/check_commits.sh

```diff
@@ -6,7 +6,7 @@ if [[ "$MESSAGE" =~ ^WIP ]]; then
 fi
 
 MAX_LENGTH=70
-TYPES="feat fix refactor perf revert chore demo docs style test"
+TYPES="build ci docs feat fix perf refactor test chore"
```

Comment:
This is the list that the changelog generation tool supports, for simplicity I want to simply stick with it :-) It is not configurable. Basically it's all minus demo/style plus build and ci. Makes sense.